app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,22 @@
+module StripeWrapper
+  class Charge
+    attr_reader :response, :error_message
+    def initialize(options={})
+      @response = options[:response]
+      @error_message = options[:error_message]
+    end
+
+    def self.create(options={})
+      begin
+        response = Stripe::Charge.create(amount: options[:amount], currency: 'usd', source: options[:source], description: options[:description])
+        new(response: response, error_message: nil)
+      rescue Stripe::CardError => e
+        new(response: e, error_message: e.message)
+      end
+    end
+
+    def successful?
+      !error_message.present?
```

Comment:
You could use `error_message.blank?` instead.

http://api.rubyonrails.org/classes/Object.html#method-i-blank-3F